pkgs/by-name/ow/owncloud-client/package.nix

```diff
@@ -2,29 +2,33 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  nix-update-script,
+  # qt
+  qt6,
+  qt6Packages,
+  # nativeBuildInputs
   pkg-config,
   cmake,
   extra-cmake-modules,
-  qt6,
-  qt6Packages,
+  # buildInputs
```

Comment:
We don't really need those comments.

Reply:
hmm, I'd keep them. I had another reviewer to explicitly ask me to keep these tidy and segmented. To whatever the convention, or lack of, I'd rather stick to it now. :)

Reply:
Sure whatever but there is no convention and I am way to lazy to maintain them so I usually nuke them